src/components/Tooltip/index.tsx

```diff
@@ -52,10 +52,14 @@ export interface StyledTooltipProps {
 }
 
 export const Tooltip: React.FC<StyledTooltipProps & TooltipProps> = ({ children, content, ...props }) => {
+  const [randomID] = useState(String(Math.random()))
+
   return (
     <>
-      <span data-tip>{children}</span>
-      <StyledTooltip type="light" effect="solid" place="bottom" {...props}>
+      <span data-tip={randomID} data-for={randomID}>
```

Comment:
Why are these random? Shouldn't they be related to the tip they are showing?

Reply:
This ID is used to distinguish each Tooltip instance. Without this all tooltips will open when one of the trigger elements are hovered.

This just has to be unique. That's why I'm just generating a random ID. Otherwise we should pass this as a prop for every instance of Tooltip, but I felt that was unnecessary. Do you feel passing a prop would be a better way?

Reply:
Ah, that makes sense. I was thinking it was for accessibility but a random number makes sense for that